onadata/apps/api/tests/viewsets/test_organization_profile_viewset.py

```diff
@@ -28,6 +29,12 @@ def setUp(self):
             'patch': 'partial_update',
         })
 
+    def tearDown(self):
+        """
+        Specific to clear cache between tests
+        """
+        cache.clear()
```

Comment:
I think we should also call `super(self.__class__, self).tearDown()`... In case the inherited class also has some custom `tearDown` functionality maybe?

Reply:
done!